pyomo/core/base/reference.py

```diff
@@ -12,6 +12,9 @@
 from pyomo.common import DeveloperError
 from pyomo.core.base.sets import SetOf, _SetProduct, _SetDataBase
 from pyomo.core.base.component import Component, ComponentData
+from pyomo.core.base.var import SimpleVar, _GeneralVarData
+from pyomo.core.base.constraint import SimpleConstraint, _GeneralConstraintData
+from pyomo.core.base.block import SimpleBlock, _BlockData
```

Comment:
With the reversion, you now have unused imports.

Reply:
Thanks